notebooks/tutorial/7 - Getting data from the ADX.ipynb

```diff
@@ -0,0 +1,417 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "ab9ec32b",
+   "metadata": {},
+   "source": [
+    "------------\n",
+    "\n",
+    "**FPS Critic Inc., makers of PureSkill.gg, is not liable for any AWS costs your incur. Run this notebook only if you understand and accept the AWS billing implications.**\n",
+    "\n",
+    "------------"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "64833230",
+   "metadata": {},
+   "source": [
+    "## Getting data from the ADX\n",
+    "\n",
+    "This notebook will help you download data from the main dataset off of the data exchange.\n",
+    "\n",
+    "Here are the definition of some of the terms around data on the AWS Data Exchange (ADX):\n",
+    "- Data Exchange: Where the data is hosted on AWS\n",
+    "- Data Product: This is our listing on the ADX where we are publishing the csds data.\n",
+    "- Data set: A container for one type of data.\n",
+    "- Revision: For the csds data, each revision is equivalent to one day of data. This allows for granularity for both data volume and date range.\n",
+    "\n",
+    "This notebook will show you how to transfer some set of revisions from the ADX to a bucket on S3. It will also allow you to enable automatic transferring for new revisions to S3."
```

Comment:
```suggestion
    "This notebook will transfer a selected set of revisions from the ADX to an S3 bucket on your AWS account. It will optionally allow you to enable automatic transfer of new revisions to S3."
```

Reply:
Done